pkg/kubectl/describe.go

```diff
@@ -927,6 +935,9 @@ func describeContainers(label string, containers []api.Container, containerStatu
 				}
 				fmt.Fprintf(out, "      %s:\t%s (%s:%s)\n", e.Name, valueFrom, e.ValueFrom.FieldRef.APIVersion, e.ValueFrom.FieldRef.FieldPath)
 			case e.ValueFrom.ResourceFieldRef != nil:
+				if nodeCapacity != nil {
+					fieldpath.MergeContainerResourceLimitsWithCapacity(&container, nodeCapacity)
```

Comment:
@sttts Your understanding is correct. 
Regarding changing output string, lets see if others have the same opinion and then I would be happy to update it. But in my view, I think it is pretty clear that any value there is upper bound anyway.